dali/operators/math/normalize/normalize.cc

```diff
@@ -63,43 +65,59 @@ has different extent across samples.
     mean = (scalar)
     stddev.shape =  [ [1, 1, 3] ] ]
 
-For color images, this normalizes the 3 color channels separately, but across all
-samples in the batch.
+For color images, this example normalizes the 3 color channels separately, but across all
+samples in the batch. This operator allows sequence inputs and supports volumetric data.
 )")
   .NumInput(1)
   .NumOutput(1)
   .SupportVolumetric()
   .AllowSequences()
-  .AddOptionalArg("batch", "If True, the mean and standard deviation are calculated across tensors "
-    "in the batch. This also requires that the input sample shapes in the non-averaged axes match.",
+  .AddOptionalArg("batch", R"code(If set to True, the mean and standard deviation are calculated
+across tensors in the batch.
+
+This argument also requires that the input sample shapes in the non-averaged axes match.)code",
     false)
-  .AddOptionalArg<float>("mean", "Mean value to subtract from the data. It can be either a scalar "
-    "or a batch of tensors with same dimensionality as the input and the extent in each dimension "
-    "must either match that of the input or be equal to 1 (in which case the value will be "
-    "broadcast in this dimension). If not specified, the mean is calculated from the input. "
-    "Non-scalar mean cannot be used when `batch` argument is True.",
-    0.0f, true)
-  .AddOptionalArg<float>("stddev", "Standard deviation value to scale the data. For shape "
-    "constraints, see *mean* argument. If not specified, the standard deviation is calculated "
-    "from the input. Non-scalar mean cannot be used when `batch` argument is True.",
+  .AddOptionalArg<float>("mean", R"code(Mean value that needs to be subtracted from the data.
```

Comment:
```suggestion
  .AddOptionalArg<float>("mean", R"code(Mean value to be subtracted from the data.
```
I don't think it _needs_ to be. We _tell it_ to be.

Reply:
Done